src/App.jsx

```diff
@@ -1,34 +1,48 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import { BrowserRouter, Outlet, Route, Routes } from 'react-router-dom'
+import Home from './pages/Home'
+import About from './pages/About'
+import Vans from './pages/Vans'
+
+import "./server"
+import VanDetails from './pages/VanDetails'
+import Layout from './components/Layout'
+import HostLayout from './components/HostLayout'
+import Income from './pages/Host/Income'
+import Reviews from './pages/Host/Reviews'
+import Dashboard from './pages/Host/Dashboard'
+import HostVans from './pages/Host/HostVans'
+import HostVanDetail from './pages/Host/HostVanDetail'
+import HostVanPricing from './pages/Host/HostVanPricing'
+import HostVanInfo from './pages/Host/HostVanInfo'
+import HostVanPhotos from './pages/Host/HostVanPhotos'
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route element={<Layout />}>
+          <Route index element={<Home />} />
+          <Route path='about' element={<About />} />
+          <Route path='vans' element={<Vans />} />
+          <Route path='vans/:id' element={<VanDetails />} />
+          <Route path="host" element={<HostLayout />}>
+            <Route index element={<Dashboard />} />
+            <Route path="income" element={<Income />} />
+            <Route path="reviews" element={<Reviews />} />
+            <Route path="vans" element={<Outlet />} >
```

Comment:
It is not necessary to add an Outlet component to this parent Route element, you can remove the element and it would still work.